src/QueryType/Update/Query/Command/Raw.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Solarium\QueryType\Update\Query\Command;
+
+use Solarium\QueryType\Update\Query\Query as UpdateQuery;
+
+/**
+ * Update query raw XML command.
+ *
+ * @see http://wiki.apache.org/solr/UpdateXmlMessages
+ */
+class Raw extends AbstractCommand
+{
+    /**
+     * XML command strings to send.
+     *
+     * @var array
+     */
+    protected $commands = [];
+
+    /**
+     * Get command type.
+     *
+     * @return string
+     */
+    public function getType(): string
+    {
+        return UpdateQuery::COMMAND_RAW;
+    }
+
+    /**
+     * Add a single XML command string to the command.
+     *
+     * @param string $command
+     *
+     * @return self Provides fluent interface
+     */
+    public function addCommand($command): self
```

Comment:
The parameter type hint should be strictly `string` unless we support other types like \DOMDocument

Reply:
I just forgot the type hint.

I don't think supporting `\DOMDocument` is necessary. Our intended use is reading existing XML from a file. If anyone wants to build the XML outside of Solarium, they can use that library's way of getting the string. That avoids introducing a dependency, albeit on a library that is enabled by default.

Reply:
Besides, if it took a `\DOMDocument`, it wouldn't be "raw".